doc/nrf/app_dev/device_guides/fem/21540ek_dev_guide.rst

```diff
@@ -52,14 +52,17 @@ For example:
 .. parsed-literal::
    :class: highlight
 
-   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dmultiprotocol_rpmsg_SHIELD=nrf21540ek
+   west build -b nrf5340dk/nrf5340/cpuapp -- -DSHIELD=nrf21540ek_fwd -Dipc_radio_SHIELD=nrf21540ek
 
-In this command, the *childImageName_* parameter has the ``multiprotocol_rpmsg_`` value and builds a multiprotocol application with support for 802.15.4 and Bluetooth.
-The *childImageName_* parameter can take the following values:
+In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds a application with support for combination of 802.15.4 and Bluetooth.
+The ``ipc_radio`` is used after build system was migrated to sysbuild. See :ref:`_child_parent_to_sysbuild_migration_sysbuild_configuration_file`
+Setting correct sysbuild option enables support for 802.15.4 and Bluetooth :ref:`ipc_radio`.
```

Comment:
```suggestion
In this command, the *childImageName_* parameter has the ``ipc_radio_`` value as default and builds the application with support for the combination of 802.15.4 and Bluetooth.
The ``ipc_radio`` has been used since the build system migration to sysbuild.
See :ref:`_child_parent_to_sysbuild_migration_sysbuild_configuration_file`.
Setting the correct sysbuild option enables support for 802.15.4 and Bluetooth :ref:`ipc_radio`.
```